### ISSUE TITLE
[new add-on] Telegram adapter

### ISSUE THREAD

**Issue by ne0bot** [mozilla-iot/addon-list#257]
This is my first project in node. It is an adapter to send Telegram notifications.
Thanks!!

```json
  {
    "name": "telegram-sender-adapter",
    "display_name": "Telegram notification Sender",
    "description": "Simple telegram sending actions. Needs to be configured with chatid and token.",
    "author": "Ne0bot",
    "homepage": "https://github.com/ne0bot/telegram-sender-adapter",
    "license": "https://github.com/ne0bot/telegram-sender-adapter/blob/master/LICENSE",
    "packages": [
      {
        "architecture": "any",
        "language": {
          "name": "nodejs",
          "versions": [
            "any"
          ]
        },
        "version": "0.0.0",
        "url": "https://github.com/ne0bot/telegram-sender-adapter/releases/download/v0.0.2-beta1/telegram-sender-adapter-0.0.2.tgz",
        "checksum": "f42c765ef7aec8b7a1a5f94751b90e15d7d54e4bdb9446a333036d9f660a249a",
        "api": {
          "min": 2,
          "max": 2
        }
      }
    ]
  }
```

**Comment by mrstegeman** [comment]
I've built a new package using this PR: ne0bot/telegram-sender-adapter#1

**Comment by ne0bot** [comment]
Update package with latest bug corrected
{
    "name": "telegram-sender-adapter",
    "display_name": "Telegram notification Sender",
    "description": "Simple telegram sending actions. Needs to be configured with chatid and token.",
    "author": "Ne0bot",
    "homepage": "https://github.com/ne0bot/telegram-sender-adapter",
    "license": "https://github.com/ne0bot/telegram-sender-adapter/blob/master/LICENSE",
    "packages": [
      {
        "architecture": "any",
        "language": {
          "name": "nodejs",
          "versions": [
            "any"
          ]
        },
        "version": "0.2.0",
        "url": "https://github.com/ne0bot/telegram-sender-adapter/releases/download/v0.0.2-beta1/telegram-sender-adapter-0.0.2.tgz",
        "checksum": "8152d2c1686ba10391872b95eb3e9ed8b78a42d5d6c3cc2f60738bc86ee21611",
        "api": {
          "min": 2,
          "max": 2
        }
      }
    ]
  }

**Comment by yalexca** [comment]
Hello ne0bot! 
I am newbie in IOT and it is my first project. I decided to use Mozilla IOT for it. The first stage of the project was successfully finished and now my three own web things work. 
On the second stage I am going to send notifications to myself using Telegram adapter. I set up Telegram bot according to the requirements and tried to set up the test rule to check this new thing.
1. I am going to set up the Rule with Telegram thing as the output. I dragged the widget to the output but was not able to open the thing property. 
2. What I have to input as the Resp - my Telegram username or my bot user name or the notification text?
3. Input thing (I use smart plug) is not able to be tied with Telegram adapter on Rules dashboard. Why?
4. Possibly I don't use the correct chatID during adapter set up procedure. Actually I used the name of my Telegram bot. Is this correct?
Thank you very much in advance, yalexca
